includes/AdminFormSave.php

```diff
@@ -106,6 +106,31 @@ public function save_rule_settings( string $form_slug ) { // phpcs:ignore
         $this->update_checkbox( Constants::MINIMUM_STOCK_THRESHOLD );
 
         $this->update_number( Constants::BELOW_STOCK_AMOUNT );
+
+        // Inside the save settings method
+        update_option(Constants::WCP_ENABLED_TAXONOMY, isset($_POST[Constants::WCP_ENABLED_TAXONOMY]) ? 1 : 0);
```

Comment:
`$this->update_checkbox( Constants::ENABLE_TAXONOMY )`
Need Changes:
* Use already in place class method `update_checkbox`. You can see its usage just above at line 106
* Update the constants name to `ENABLE_TAXONOMY`
* Kindly remember that the name of the field should also be `Constants::ENABLE_TAXONOMY`